demux/demux_lavf.c

```diff
@@ -1005,6 +1005,9 @@ static int demux_open_lavf(demuxer_t *demuxer, enum demux_check check)
                    "analyzeduration to %f\n", analyze_duration);
     }
 
+    if (matches_avinputformat_name(priv, "hls"))
```

Comment:
This should go into `format_hacks` list.

Reply:
I hadn't put it there because most format_hacks are also applicable to other demuxers, while this option only exists for the hls demuxer